electron/main.ts

```diff
@@ -320,6 +330,7 @@ const signIn = async (event, userData) => {
                 }
             })
             .catch((error) => {
+                console.error(error.message);
```

Comment:
mixing console logging and winston logging here - is that on purpose or just accidentally left in?